pkg/vm/stack_item.go

```diff
@@ -857,3 +858,78 @@ func (i *InteropItem) Convert(typ StackItemType) (StackItem, error) {
 func (i *InteropItem) MarshalJSON() ([]byte, error) {
 	return json.Marshal(i.value)
 }
+
+// PointerItem represents VM-level instruction pointer.
+type PointerItem struct {
+	pos  int
+	hash util.Uint160
```

Comment:
That won't work long-term, sure it's enough for preview2, but the whole point of these pointers it to be able to pass them around between contexts, so we need to have the associated script in this structure.

Reply:
I will use both for now. Some optimizations can be made if this becomes critical.